precheck/lib/precheck/options.rb

```diff
@@ -90,19 +90,19 @@ def self.available_options
                                      short_option: "-r",
                                      env_name: "PRECHECK_DEFAULT_RULE_LEVEL",
                                      description: "The default rule level unless otherwise configured",
-                                     is_string: false,
+                                     skip_type_validation: true, # allow String, Symbol
```

Comment:
This can be a `Symbol` type! It will auto convert a string to a symbol

```suggestion
                                     type: Symbol
```